src/Statues/slavers.js

```diff
@@ -0,0 +1,10 @@
+const SlaverStatues = [ 
+  {
+    statueName: "Edward Colston"
```

Comment:
As the object is named statue, I don't think each element needs it. So `name` is fine.